mocha-client-tests/tests/lib/background.test.js

```diff
@@ -0,0 +1,9 @@
+describe('Background', function() {
+    describe('ping', function() {
+        it('should return pong in response', function() {
+            Background.ping(false, false, function(response) {
+                response.should.equal('pong');
```

Comment:
chai's prototype hack might be confusing to the casual JavaScript reader. Could you update this to use [expect()](http://chaijs.com/api/bdd/) (or something) instead? It would look like `expect(response).to.equal('pong')`